.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Build
+      run: |
+        export PATH=$HOME/go/bin:$PATH
+        go get -u golang.org/x/lint/golint
```

Comment:
I'd love to run linter locally and do quick check before I open a PR. What about adding `make setup` & `make lint` goal and refer to them in here? We did similar thing [here](https://github.com/aquasecurity/harbor-scanner-trivy/blob/master/Makefile#L26).